src/client/common/client_common.cpp

```diff
@@ -91,3 +94,25 @@ std::unique_ptr<mp::SSLCertProvider> mp::client::get_cert_provider()
     auto client_cert_dir = mp::utils::make_dir(data_dir, "client-certificate");
     return std::make_unique<mp::SSLCertProvider>(client_cert_dir);
 }
+
+void mp::client::set_logger()
+{
+    set_logger(mpl::Level::info);
+}
+
+void mp::client::set_logger(mpl::Level verbosity)
+{
+    mpl::set_logger(std::make_shared<mpl::StandardLogger>(verbosity));
+}
+
+void mp::client::preliminary_setup()
+{
+    try
+    {
+        mp::platform::preliminary_gui_autostart_setup();
+    }
+    catch (std::runtime_error& e)
+    {
+        mpl::log(mpl::Level::warning, "client", e.what());
```

Comment:
This should probably be an error, no?

Reply:
Sure! I thought you wouldn't like that.

Reply:
Yeah it ultimately means there's a problem with how the app is being run / packaged, so this should be and error, IMO.

Reply:
Done.